### PR TITLE
db-console: update notification icon in alert box to use new cockroach logo

### LINKED ISSUE #127829: ui: cockroach icon used in db console upgrade box is out of date
We have a newer cockroach logo that should replace the old one.

image (5)

Jira issue: CRDB-40627

### PR DESCRIPTION
**Pull request by angles-n-daemons**
The current icon for the notification alert is the outdated cockroach db logo. To make it consistent with the rest of the system, it needed to be updated. This change both removes the old logo from the codebase as well as uses the new one from our shared icon library.

Epic: none
Fixes: #127829

Release note (ui change): changes notification alert icon

### CODE REVIEW

File: pkg/ui/workspaces/db-console/src/views/shared/components/alertBox/index.tsx [hunk 1]
```diff
@@ -12,23 +12,27 @@ import classNames from "classnames";
...
   warningIcon,
-  notificationIcon,
   criticalIcon,
 } from "src/views/shared/components/icons";
 
+import { CockroachMarkLightFull } from "@cockroachlabs/icons";
+
 import "./alertbox.styl";
 
+export function alertIconFromSVG(svg: string): JSX.IntrinsicElements["img"] {
```
[the remark this answers is not in the document]
Comment (angles-n-daemons) [reply pkg/ui/workspaces/db-console/src/views/shared/components/alertBox/index.tsx:24]: that's not a bad idea, I'm going to see if I can inline these all as react components instead

File: pkg/ui/workspaces/db-console/src/views/shared/components/icons/index.tsx [hunk 2]
```diff
@@ -58,63 +39,102 @@ export const rightArrow = `<svg width="7px" height="10px" viewBox="0 0 7 10" ver
     </g>
 </svg>`;
 
-export const criticalIcon = `<svg width="20px" height="20px" viewBox="0 0 20 20" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 42 (36781) - http://www.bohemiancoding.com/sketch -->
+export const criticalIcon = (
```
Comment (angles-n-daemons) [inline pkg/ui/workspaces/db-console/src/views/shared/components/icons/index.tsx:42]: These icons change the string literal svgs to inline jsx elements

**Review by xinhaoz — APPROVED** [review]

**Pull request merged**